huggingface/tensorflow/training/docker/2.6/py3/cu112/Dockerfile.gpu

```diff
@@ -2,7 +2,18 @@
 # refer to the above page to pull latest Tensorflow image
 
 # docker image region us-west-2
-FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/tensorflow-training:2.6.3-gpu-py38-cu112-ubuntu20.04
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/tensorflow-training:2.6.3-gpu-py38-cu112-ubuntu20.04 AS base_image
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib"
+
+RUN apt-get update \
+ && apt-get upgrade -y \
+ && apt-get autoremove -y \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+FROM base_image AS ec2
```

Comment:
We do not want this change. Please revert it.

Reply:
Thanks for checking. After reverting this, do I also comment the build portion for this image in the buildspec file?